### PR TITLE
Add project ipfsMart

### PR DESCRIPTION
**Pull request by luowenw**

### CODE REVIEW

**Comment by luowenw** [comment]
Add project ipfsMart

**Comment by dineshshenoy** [comment]
Can you please provide your submission ID from your registration?  (https://slingshot.filecoin.io/register-now).  It should be at the top of the confirmation email.

**Comment by luowenw** [comment]
> Can you please provide your submission ID from your registration? (https://slingshot.filecoin.io/register-now). It should be at the top of the confirmation email.

Hi, I just submit the form and the submission ID is 675236401, please take a look, thanks!

File: participants/ipfsMart.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+We wish to contribute to 0.5-1T quantity of data.   
+
+(7) How are you structuring the data?   
+Most of these data is unstructured, like videos and images.   
+
+(8) What pre-processing are you doing before ingesting the data?   
+We transfer user's data, intermediate models and final models/applications to the IPFS network and assign a unique identifier for each piece of data.   
+
+(9) What tech stack will you use to store the data?   
+lotus
```
Comment (dineshshenoy) [inline participants/ipfsMart.md:36]: Just confirming that you'll be deploying on the Filecoin network (a lot of the description above was about IPFS).  Regardless IPFS may be a great caching layer for hot retreival.

**Review by dineshshenoy — APPROVED** [review]

**Pull request merged**